pg_search/src/postgres/customscan/pdbscan/exec_methods/fast_fields/string.rs

```diff
@@ -103,11 +103,16 @@ impl ExecMethod for StringFastFieldExecState {
             match self.search_results.next() {
                 None => ExecState::Eof,
                 Some((scored, doc_address, term)) => {
+                    let heaprel = self
+                        .inner
+                        .heaprel
+                        .as_ref()
+                        .expect("MixedFastFieldsExecState: heaprel should be initialized");
```

Comment:
```suggestion
                        .expect("StringFastFieldExecState: heaprel should be initialized");
```
... am hoping to merge these soon.